packages/solana/lib/src/encoder/message/decode_message_args.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:solana/src/encoder/address_lookup_table/address_lookup_table.dart';
+import 'package:solana/src/encoder/message/loaded_addresses.dart';
+
+part 'decode_message_args.freezed.dart';
+
+@freezed
+class DecodeMessageArgs with _$DecodeMessageArgs {
+  const factory DecodeMessageArgs.accountKeys(
```

Comment:
Where does it come from at all? I don't see any usages of it. What is the use case?

Reply:
This union class is used to decode the transaction, it is used here: https://github.com/espresso-cash/espresso-cash-public/blob/241d8930ffb5dceb89bc2bd57a2f156b7094ca5b/packages/solana/lib/src/encoder/signed_tx.dart#L69